.github/workflows/docker-release-image.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted-docker
```

Comment:
@bdevans does this look appropriate? There is a dedicated runner tagged with this label, but are there other changes likely to be needed? I'm not sure we can currently test this workflow without merging.